src/elements/core/styles/mixins/table.scss

```diff
@@ -0,0 +1,135 @@
+@use './typo' as t;
+
+/*
+ * SBB table mixin
+ * Notes:
+ * We cannot use `border-collapse` because it is not compatible with the `border-radius` property
+ */
+@mixin table {
+  @include table-striped;
+  @include table-m;
+
+  --sbb-table-border: 1px solid var(--sbb-table-border-color);
+  --sbb-table-border-color: var(--sbb-color-cloud);
+  --sbb-table-border-radius: var(--sbb-border-radius-4x);
+  --sbb-table-background-color: var(--sbb-color-white);
+  --sbb-table-row-striped-color: var(--sbb-color-milk);
+  --sbb-table-color: inherit;
+  --sbb-table-caption-color: var(--sbb-color-granite);
+  --sbb-table-caption-margin-block-start: var(--sbb-spacing-fixed-4x);
+
+  background-color: var(--sbb-table-background-color);
+  border: var(--sbb-table-border);
+  border-radius: var(--sbb-table-border-radius);
+  border-spacing: 0;
+  caption-side: bottom;
+  color: var(--sbb-table-color);
+  table-layout: auto;
+
+  thead {
+    & > tr > th {
+      @include table-header-cell;
+    }
+  }
+
+  tbody {
+    & > tr {
+      @include table-data-row;
+
+      & > td {
+        @include table-data-cell;
+      }
+    }
+  }
+
+  tfoot {
+    // TBD
+  }
+
+  caption {
+    @include table-caption;
+  }
+}
+
+@mixin table-header-cell {
+  @include t.text-xs--bold;
+
+  border-bottom: var(--sbb-table-border);
```

Comment:
Is it possible to use `border-block-end` and `border-inline-end` ? Also for similar following occurrences

Reply:
I agree, here it doesn't matter. I was just thinking at our base rules to everywhere use "modern" properties. In general I'd like to not have a border-radius and user border-collapse :D... But let's start with how it was designed and we will see how far we will bring it with all future table features.